lib/thumbnail-sizes.php

```diff
@@ -8,5 +8,7 @@
   add_image_size( 'admin-thumb', 150, 150, false );
   add_image_size( 'opengraph', 1200, 630, true );
 
+  add_image_size( 'col-4', 1000, 9999, false );
```

Comment:
this should match our column sizing class syntax. `item-m-4` or whatever it is related. with the breakpoint indicated.